src/nyc_trees/apps/core/templates/base.html

```diff
@@ -50,6 +55,30 @@
                         | <a href="{% url 'registration_register' %}">Register</a>
                         {% endif %}
                     </div>
+                    {% if my_events_now %}
+                    <div class="nav-right">
+                        <div class="dropdown">
+                            <button class="btn btn-default dropdown-toggle" type="button" id="eventNowAlert" data-toggle="dropdown" aria-expanded="false">
+                                !
+                            </button>
+                            <ul class="dropdown-menu dropdown-menu-right" role="menu" aria-labelledby="eventNowAlert">
+                                {% for event in my_events_now %}
+                                <li role="presentation">
+                                    <a role="menuitem" tabindex="-1" href="{{ event.get_user_checkin_url }}">
+                                        The event
+                                        {{ event.title }}
+                                        {% if event.has_started %}
+                                            has started
```

Comment:
Could not see the "has started" message even when I was RSVPed to an event that was in progress.